### DISCUSSION TITLE
1.1.1.1

### DISCUSSION THREAD

**Discussion by Pirih61** [XIU2/CloudflareSpeedTest#327] [讨论]
Speed

**Comment by XIU2** [comment]
？

**Comment by Pirih61** [comment]
it accelerates in the morning, my internet speed does not see 1 mbps at night, it is very slow

**Comment by Pirih61** [comment]
My download speed is 100mbs at 5 am, while my download speed is 1 mbs at 10 am is ridiculous.

**Comment by XIU2** [comment]
This situation is very common.

**Comment by Pirih61** [comment]
Well, will it be fixed? I need internet speed in the evening. I'm at 100 mbs at 5 am. I installed 1.1.1.1 to speed it up, it only provides security

**Comment by Pirih61** [comment]
Can I tell you my problems that are not fast
